### PR TITLE
Allow Bounds to be a number[]

### PR DESCRIPTION
**Pull request by vespakoen**

### CODE REVIEW

**Comment by pravdomil** [comment]
can you please send me example code where type `Bounds` are used?

**Comment by vespakoen** [comment]
```js
const bounds: Bounds = [0, 0, 200, 200]
panel = new Window('dialog', 'Starter Panel', bounds, {
  closeButton: true,
  resizeable: true
})
```

File: ScriptUI/index.d.ts [hunk 1]
```diff
@@ -4187,6 +4187,8 @@ declare class Bounds {
 
 }
 
+declare type Bounds = ObjectBounds | number[]
```
Comment (pravdomil) [inline ScriptUI/index.d.ts]: please add semicolon

**Comment by vespakoen** [comment]
This should not be used, Bounds cannot be an Object, it should always be an array, see my other PR which fixes this better.

**Comment by pravdomil** [comment]
```js
var bounds = [0, 0, 200, 200];
panel = new Window('dialog', 'Starter Panel', bounds, {
  closeButton: true,
  resizeable: true
})

alert(panel.bounds[2]); // 200
alert(panel.bounds.width); // 200
```

both works

**Comment by vespakoen** [comment]
Good to know!

**Comment by pravdomil** [comment]
should I merge it?

**Comment by vespakoen** [comment]
No, the number[] is incorrect, I'll fix it in another pr, it has to be something like:

```js
type ArrayBounds = [number, number, number, number];
```

**Pull request closed without merging**